internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixReadOnlyStoreRepositoryAdapter.java

```diff
@@ -84,6 +84,11 @@ public Store getStoreOrThrow(String storeName) throws VeniceNoStoreException {
     return store;
   }
 
+  // test only
+  public Set<StoreDataChangedListener> getListeners() {
```

Comment:
Same here? Can we make it package private and use `Collections::unmodifiableSet`?